pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -20,3 +29,66 @@ class FileRepository(Repository):
     )
 
     mutable_urls = attr.ib(default=attr.Factory(lambda: ["PULP_MANIFEST"]), type=list)
+
+    def upload_file(self, file_obj, relative_url=None):
+        """Upload a file to this repository.
+
+        Args:
+            file_obj (str, file object)
+                If it's a string, then it's the path of file to upload.
+                Else, it ought to be a file-like object
```

Comment:
Might be nice to link to https://docs.python.org/3/glossary.html#term-file-object

Reply:
Also, could you please stick with the style of using full sentences (so, end them with a full stop ".") ?